app/src/main/res/values/strings.xml

```diff
@@ -1274,7 +1274,7 @@
     <string-array name="locale_credits" translatable="false">  <!-- one-to-one with locale_display -->
         <item />
         <item><![CDATA[<a href=https://github.com/Verdulo>Verdulo</a>]]></item>
-        <item><![CDATA[<a href=https://github.com/Jej>Jej</a>|<a href=https://github.com/Aloha>Aloha</a>]]></item>
+        <item><![CDATA[<a href=https://github.com/Jej>Jej</a>|<a href=https://github.com/Aloha>Aloha</a>|<a href=https://github.com/ldmpub>ldmpub</a>]]></item>
```

Comment:
The locale_credits are displayed in the About dialog (also copied to places like the README, etc). 

I added something in the appropriate location - put whatever you want here.

Reply:
Ok I'm not familiar with claiming any credit for my translations .. do it your own way 😎.

Reply:
I like to keep track of attribution like this, but its totally optional - link to your github, no link, full name, pseudonym, or no entry at all - whatever - please don't hesitate to change this value.